qiskit/visualization/pass_manager_visualization.py

```diff
@@ -31,8 +31,8 @@ def pass_manager_drawer(pass_manager, filename=None, style=None, raw=False):
     """
     Draws the pass manager.
 
-    This function needs `pydot <https://github.com/erocarrera/pydot>`__, which in turn needs
-    `Graphviz <https://www.graphviz.org/>`__ to be installed.
+    This function needs `pydot <https://github.com/pydot/pydot>`_, which in turn needs
+    `Graphviz <https://www.graphviz.org/>`_ to be installed.
```

Comment:
The double underscore is also fine here - the single creates a named hyperlink (so you could later use `<Graphviz_>` as if it was the URL and it'd alias the first one), while the double doesn't allow the re-use.  The docs are here: https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases.

No need to change anything here - both are fine - just pointing out that you don't need to worry about this in the future.

Reply:
always learning new things. reverting in c6af56cb58ac9edbbbea7da9e9dd7b72d7b48e84 so the diff is cleaner.